src/handlers/comment/handlers/index.ts

```diff
@@ -42,12 +43,37 @@ export const issueClosedCallback = async (): Promise<void> => {
   const issue = (_payload as Payload).issue;
   try {
     const comment = await handleIssueClosed();
+    await addCommentToIssue(
+      `If you enjoy the DevPool experience, please follow <a href="https://github.com/ubiquity">Ubiquity on GitHub</a> and star <a href="https://github.com/ubiquity/devpool-directory">this repo</a> to show your support. It helps a lot!`,
+      issue!.number
+    );
     return await addCommentToIssue(comment!, issue!.number);
   } catch (err: any) {
     return await addCommentToIssue("Error: " + err.message, issue!.number);
   }
 };
 
+/**
+ * Callback for issues created - Processor
+ */
+
+export const issueCreatedCallback = async (): Promise<void> => {
+  const { payload: _payload } = getBotContext();
+  const config = getBotConfig();
+  const issue = (_payload as Payload).issue;
+  const labels = issue!.labels;
+  try {
+    const timeLabels = config.price.timeLabels.filter((item) => labels.map((i) => i.name).includes(item.name));
+    const priorityLabels = config.price.priorityLabels.filter((item) => labels.map((i) => i.name).includes(item.name));
+
+    if (timeLabels.length === 0) await createLabel("Time: <1 Hour");
+    if (priorityLabels.length === 0) await createLabel("Priority: 0 (Normal)");
```

Comment:
The default labels here should load from our config actually. If there is nothing in the config then this should not do anything.

Reply:
yeah, there is no problem. But what is the variable name? I mean, I can I get the data?

For example, config.default_timeLabel, config.default_priorityLabel???

Reply:
Yeap! I've fixed it by picking up the lowest price label.